images/builder/install-builder-deps.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Copyright 2017-2020 Authors of Cilium
+# SPDX-License-Identifier: Apache-2.0
+
+set -o xtrace
+set -o errexit
+set -o pipefail
+set -o nounset
+
+packages=(
+  libelf1
+  libmnl0
+  bash-completion
+  iptables
+  kmod
+  ca-certificates
+)
+
+apt-get update
+
+ln -fs /usr/share/zoneinfo/UTC /etc/localtime
```

Comment:
what's this for?

Reply:
For some reason new version of Ubuntu requires user to set timezone for some packages, I don't know why.

Reply:
My guess is that from Ubuntu distro point of view, there is no such thing as default timezone any more.

Reply:
I've added a comment above, it actually got me puzzled when I first noticed it in a Dockerfile that Daniel wrote.